CONTRIBUTING.md

```diff
@@ -1,172 +1,120 @@
-                                                  +--------------------------+
-                                                  |
-                                                  |
-+-----------------------+                         |
-|                       |                 +-------+-------+            +-------------------+
-|       Sonatype        |   upload poms   |               |  search    |                   |
-|                       +---------------> |   Scaladex    +--------->  |   Elasticsearch   |
-|  +-----------------+  |                 |      Web      |            |                   |
-|  |                 |  |  edit projects  |               |            +-------------------+
-|  | Scala Developer +------------------> +-------+-------+
-|  |                 |  |                         |
-|  +-------------+---+  |                         | read claims   +------------------------+
-|                |      |                         +------------>  |                        |
-+-----------------------+                                         | [scaladex-contrib.git] |
-                 |                 claims projects                |                        |
-                 +--------------------------------------------->  +------------------------+
-
--- asciiflow.com
+sbt:scaladex> data/run init
 ```
-
-As shown in the above diagram, Scaladex uses two Git repositories 
-([scaladex-contrib.git](https://github.com/scalacenter/scaladex-contrib) and [scaladex-index.git](https://github.com/scalacenter/scaladex-index)) as persistence mechanisms, 
-and ElasticSearch to power the full-text search.
-
-The `contrib.git` repository is read-only for the Scaladex application
-(users can write on it, on Github).
-
-Data of the `[scaladex-index.git]` repository are written *only* by Scaladex.
-This repository contains the POMs and project information added by users.
-
-## Bintray Data Pipeline
-
-If you want to update the data:
-
-The entry point is at [data/Main.scala](/data/src/main/scala/scaladex/data/Main.scala)
-
-### List Poms
-
-This step searches on Bintray for artifact containing `_2.10`, `_2.11`, `_2.12`, ...
-Bintray contains jcenter, it's a mirror of maven central.
-
-You will need a premium Bintray account.
-
+2. Then you can start the server with:
 ```
-set javaOptions in reStart ++= Seq(
-  "-DBINTRAY_USER=XXXXXXX", 
-  "-DBINTRAY_PASSWORD=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-)
-
-data/reStart list
+sbt:scaladex> server/run
 ```
+3. Finally you can open the website in your browser at `localhost:8080`.
 
-### Download ivy.xml of SBT plugins from Bintray
-
-This step also requires a premium Bintray.
+The server will start fetching information from Github progressively, you will see them appear after some time.
 
-```
-data/reStart sbt
-```
+After you stop and start the server again, the github information will still be there.
```

Comment:
because there are store in this directory locally ?

Reply:
:+1: